### DISCUSSION TITLE
Is Solnet dead?

### DISCUSSION THREAD

**Discussion by ghost** [bmresearch/Solnet#463] [Q&A]
Hi there! 
Solnet is an absolute great tool! Unfortunately, it seems that here nobody is active anymore and the project isn't maintained any longer.
And the site blockmountain.io is not reachable since a while, too.
What happened to this great project? Is there still some actitity? Are there alternatives for C# programmers who want to deal with solana? What are the plans? Does using Solnet make sense, or is it to be regarded as "deprecated"?

**Comment by ijuedt** [comment]
Unfortunately, that's my impression, too: Solnet isn't being developed any more. I'd like to use Solana, but from a C# client's perspective, I can only rely on Nethereum.

**Comment by ghost** [comment]
Yep. That's a pity. And even more disappointing is that the owner/starter of this project does not mention anything to the current state of Solnet.

Solana is my first target for programming, too. Without Solnet, my first try will be to call the Solana APIs directly within a helper method, which creates a Process for cmd.exe (on a Windows client) and run's a call. After finishing, I'll have to sort out the process output to get the data I need. Alternatively, I'll try to embed some Rust calls and function in my C# program. But all this is complicated and annoying. Solnet would have been a much better choice.

I considered Nethereum too in the past, but for doing some ETH programming. That it supports Solana is new to me, are you sure and if so, can you give more information?

**Reply by ijuedt** [reply]
This is a misunderstanding, unfortunately. My requirement is unspecific enough (programmable blockchain) for Ethereum to be sufficient. I would have liked to use Solana, because I expected to be able to use Rust instead of Solidity. But I can do without it. So I'm sorry I can't help you here.

**Reply by ghost** [reply]
NP. I'll follow the advice of BifrostTitan (Thank you!), and will try to build from source, as soon as I'll find time.

**Comment by BifrostTitan** [comment]
Solnet is actively maintained by the community now. Use the NET8 tag to find the latest nuget packages or build from source.
The current state of solnet is used in both the Unity & Unreal Engine SDKs. If you are having trouble learning how to use the SDK just ask co-pilot for help.
